src/utils/components/ExternalLink/ExternalLink.tsx

```diff
@@ -18,14 +19,14 @@ const ExternalLink: React.FC<ExternalLinkProps> = ({
   stopPropagation,
 }) => (
   <a
-    className={classNames('co-external-link', additionalClassName)}
+    className={additionalClassName}
```

Comment:
+1 to the `co-...` classname removal, can you also rename `additionalClassName` to be `className`?